circle.yml

```diff
@@ -42,7 +42,7 @@ dependencies:
     - ~/.yarn
   override:
     - yarn install
-    - yarn global add flow-coverage-report
+    - yarn global add flow-coverage-report@0.3.0
```

Comment:
Circle would fail to find a version of `flow-coverage-report` and fail out. 🙁 When I ran this locally it found the version. I noticed a couple other PRs also had this failure though.